src/abaqus/BasicGeometry/EdgeArray.py

```diff
@@ -128,7 +128,8 @@ def findAt(self, *args, **kwargs) -> Union[Edge, List[Edge]]:
             An :py:class:`~abaqus.BasicGeometry.Edge.Edge` object or a sequence of Edge objects.
 
         """
-        return Edge() if isinstance(args[0],float) else [Edge()]
+        first_value = kwargs.get('coordinates') if 'coordinates' in kwargs else args[0]
+        return Edge() if isinstance(first_value,float) else [Edge()]
```

Comment:
This first line can actually be simplified to `first_value = kwargs.get('coordinates', args[0])`.

Can you add a space after the comma inside the `instance` function?

Reply:
Right!

Reply:
This modification was making the test not pass

(because it tries to access `args[0]` when `args` is `()`)

And I think the implementation was wrong, I corrected it now

Reply:
How about `first_value = kwargs.get('coordinates', args[0] if args else ())`?

Reply:
Great!